Lib/test/test_frozen.py

```diff
@@ -7,23 +7,36 @@
 #
 # The test_importlib also tests this module but because those tests
 # are much more complicated, it might be unclear why they are failing.
-# Invalid marshalled data in frozen.c could case the interpreter to
-# crash when __hello__ is imported.
 
 import sys
 import unittest
-from test.support import captured_stdout
+from test.support import captured_stdout, impl_detail
 
 
 class TestFrozen(unittest.TestCase):
     def test_frozen(self):
         name = '__hello__'
         if name in sys.modules:
             del sys.modules[name]
+        # Invalid marshalled data in frozen.c could case the interpreter to
+        # crash when __hello__ is imported.  Ensure we can import the module
+        # and that it generates the correct output.
         with captured_stdout() as out:
             import __hello__
         self.assertEqual(out.getvalue(), 'Hello world!\n')
 
+    @impl_detail('code object line table', cpython=True)
```

Comment:
a small nitpick: cpython arg is not needed and other uses of @impl_detail in tests leave it out (with just one exception).